### PR TITLE
Draft: initial commit for web scraping program using Selenium

### PR DESCRIPTION
**Pull request by KushRaghani**

### CODE REVIEW

File: web-scraper/index.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+from selenium import webdriver
+from bs4 import BeautifulSoup
+import re
+
+#Import configuration data
+from settings import LINKEDIN_USERNAME,LINKEDIN_PASSWORD,LINKEDIN_LOGIN_URL,LINKEDIN_POSTS_URL,POSTS_SEARCH_TERMS,CHROME_DRIVER_PATH
+
+#Initialize Selenium web driver for Chrome
+browser = webdriver.Chrome(CHROME_DRIVER_PATH)
```
Comment (anoop1409) [inline web-scraper/index.py:9]: We can get rid of the `CHROME_DRIVER_PATH` completely. It should work out of the box if chromedriver is installed and added to the PATH.

**Comment by thisischandanmishra** [comment]
Code looks good to me. I can start working on my part once code is merged.

File: web-scraper/index.py (hunk 1 above)
Comment (thisischandanmishra) [reply web-scraper/index.py:9]: Yes, I can install it on the VM at boot time even if it's not found. So no worries.

**Review by anoop1409 — APPROVED** [review]
Approving the initial MR for allowing others to continue the work

**Pull request merged**